services/QuillLMS/spec/factories/prompt_health.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
```

Comment:
I would avoid magic numbers in specs and instead generate random ones.  You can use [faker](https://github.com/faker-ruby/faker/blob/master/doc/default/number.md#fakernumber) to populate various ones with conditions.